README.md

```diff
@@ -1,135 +1,223 @@
+        notify:
+            - devops-alert-pagerduty
+        assertions:
+            - metric.duration < 5 minutes
+
+    send-welcome-email:
+        schedule: every 10 minutes
+        assertions:
+            - metric.count > 0
+            - metric.duration < 30 seconds
+
+# configure all of your monitors with type "synthetic"
+synthetics:
+    cronitor-homepage:
+        request:
+            url: https://cronitor.io
+            regions:
+                - us-east-1
+                - eu-central-1
+                - ap-northeast-1
+        assertions:
+            - response.code = 200
+            - response.time < 2s
+
+    cronitor-telemetry-api:
+        request:
+            url: https://cronitor.link/ping
+        assertions:
+            - response.body contains ok
+            - response.time < .25s
+
+events:
+    production-deploy:
+        notify:
+            alerts: ['deploys-slack']
+            events: true # send alert when the event occurs
 
-Cronitor does not require a token for pinging a monitor unless you have enabled Ping API authentication in your account settings. At the moment, this gem does not support Ping API auth.
+```
 
-In that case:
 
-```ruby
-my_monitor = Cronitor.new code: 'abcd'
-```
+## Contributing
 
-The aforementioned ping methods can now be used.
+Pull requests and features are happily considered! By participating in this project you agree to abide by the [Code of Conduct](http://contributor-covenant.org/version/2/0).
 
-## Development
+### To contribute
 
-After checking out the repo, run `bin/setup` to install dependencies. Then, run `rake spec` to run the tests. You can also run `bin/console` for an interactive prompt that will allow you to experiment.
+Fork, then clone the repo:
 
-To install this gem onto your local machine, run `bundle exec rake install`. To release a new version, update the version number in `version.rb`, and then run `bundle exec rake release`, which will create a git tag for the version, push git commits and tags, and push the `.gem` file to [rubygems.org](https://rubygems.org).
+    git clone git@github.com:your-username/cronitor-ruby.git
 
-## Contributing
 
-1. Fork it ( https://github.com/evertrue/cronitor/fork )
-2. Create your feature branch (`git checkout -b my-new-feature`)
-3. Commit your changes (`git commit -am 'Add some feature'`)
-4. Push to the branch (`git push origin my-new-feature`)
-5. Create a new Pull Request
+Set up your machine:
+
+    bin/setup
+
+
+Make sure the tests pass:
+
+    rake spec
+
+
+Make your change. You can experiment using:
+
+    bin/console
+
+
+Add tests for your change. Make the tests pass:
+
+    rake spec
+
+Push to your fork and [submit a pull request]( https://github.com/cronitorio/cronitor-ruby/compare/)
+
 
 ## Release a new version
 
-The `bump` gem makes this easy:
+The bump gem makes this easy:
 
-1. `rake bump:(major|minor|patch|pre)`
-2. `rake release`
+1. rake bump:(major|minor|patch|pre)
+2. rake release
```

Comment:
Any reason why the `code` formatting was removed? These are CLI commands, after all.

Reply:
oops, not intentional!